_posts/2015-04-27-durham-restaurant-time-machine.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: "Building a time machine for Durham's food landscape"
+date: 2015-04-27
+author: Tim Stallmann
+tags: cartography leaflet durham
+summary: Remember *Another Thyme*? *Kim Son*? *Honey's*? The *Know* restaurant and book store? As Durham gains more notoriety for its "foodie scene," it's easy to forget restaurants of the past. So here at Savas, we've created a Durham restaurant time machine.
+---
+Remember *Another Thyme*? *Kim Son*? *Honey's*? The *Know* restaurant and book store? As Durham gains more notoriety for it's "foodie scene," it's easy to forget restaurants of the past.
+So here at Savas, we've created a helpful [Durham restaurant time machine](http://www.savaslabs.com/durham-restaurants-map).
+
+<div class="blog-image wrap-left">
+<img src="/assets/img/blog/restaurants-map-pan-pan-diner.png" alt="Map showing the Pan Pan Diner, at its original location on Hillandale Road" width="250" height="212">
+<p>The Pan Pan Diner, at its original location on Hillandale Road (misspelled as <i>Hillondale</i> in the health inspections dataset)</p>
+</div>
+
+With this tool, you can click back through the years (from 1995 through 2015) to see which restaurants were open when, and where.
+No longer do you have to wonder:
+
+* Where was the original Pan Pan Diner located? (Hillandale Rd and I-85)
+* Where was Ninth Street Bakery before it was on W Chapel Hill Street, and when did it move (776 Ninth Street, 1997)
+* Is it true what people say that there were no restaurants downtown in the 1990s? (not exactly, but there were a lot fewer)
+
```

Comment:
@timstallmann - should this say "Is it true when people say"? I'm honestly not sure.